integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/LRAParticipantTestResourceLifecycle.java

```diff
@@ -22,7 +22,7 @@ public static String getCoordinatorEndpoint() {
 
     @Override
     public Map<String, String> start() {
-        registry = new GenericContainer<>("jbosstm/lra-coordinator:5.12.0.Final")
+        registry = new GenericContainer<>("quay.io/jbosstm/lra-coordinator:latest")
```

Comment:
Can't say I like `latest`, but let's go with it for now

Reply:
Yeah, I decided to make things consistent between this and the MP tests. If it becomes a problem, we can drop it easily.

Reply:
👍